lib/mpl_toolkits/axisartist/grid_finder.py

```diff
@@ -219,30 +219,20 @@ def __init__(self, nbins=10, steps=None,
         super().__init__(nbins, steps=steps, integer=integer,
                          symmetric=symmetric, prune=prune)
         self.create_dummy_axis()
-        self._factor = 1
 
     def __call__(self, v1, v2):
-        locs = super().tick_values(v1 * self._factor, v2 * self._factor)
-        return np.array(locs), len(locs), self._factor
-
-    @_api.deprecated("3.3")
-    def set_factor(self, f):
-        self._factor = f
+        locs = super().tick_values(v1, v2)
+        return np.array(locs), len(locs), 1  # 1: factor
```

Comment:
added a ref to angle_helper
"factor" is not really documented, but is basically a way for axisartist angular locators to return locations in minutes and seconds (of angle) instead of degrees, by setting the factor to 60/3600.